lib/cdo/shared_constants.rb

```diff
@@ -38,6 +38,13 @@ module SharedConstants
     }
   ).freeze
 
+  # The set of default values for a section
+  SECTION_DEFAULTS = OpenStruct.new(
```

Comment:
I wasn't entirely sure how to use this since it's in `lib/cdo/shared-constants.rb` and I'd like to avoid hardcoding the string in the places above. Any suggestions?

Reply:
1.) You'll need to add `SECTION_DEFAULTS` to the list in  the script that generates shared constants, https://github.com/code-dot-org/code-dot-org/blob/aa6a61a400862c70b1de0fb03e38d644e51621b4/apps/script/generateSharedConstants.rb#L73. 
2.) Then, you should have access to your constant with this pattern: https://github.com/code-dot-org/code-dot-org/blob/06ae58fdd52a97d779aff24db753dabc8d4afd97/pegasus/helpers/section_api_helpers.rb#L256